app/assets/stylesheets/pages/_static-pages.scss

```diff
@@ -42,4 +46,4 @@
   img {
```

Comment:
Would be great to not target any elements with tag names and only use classes... Not sure how much code like that is there left? Or is it because the image is customisable and can't have a class?